docs/operations/fencing.md

```diff
@@ -1,57 +1,51 @@
 ---
-title: "Fencing"
-linkTitle: Fencing
+title: "How To Enable Fencing"
+linkTitle: How To Enable Fencing
 ---
 
-For information regarding the Ondat Fencing feature, see our
-[fencing concepts page](/docs/concepts/fencing).
+## Overview
 
-## Labeling a pod for fencing
+> 💡 For more information on the Ondat fencing feature, review the [Fencing](/docs/concepts/fencing) feature page.
 
-When Ondat detects that a node has gone offline or become partitioned, it
-marks the node offline and performs volume failover operations.
+### How To Label a Pod for Fencing?
 
-The [Ondat Fencing Controller](https://github.com/storageos/api-manager/tree/master/controllers/fencer)
-watches for node failures and determines if there are any pods targeted for
-fencing. In order for a pod to be fenced, the following criteria is required.
+When Ondat detects that a node has gone offline or become partitioned, it marks the node offline and performs volume failover operations.
+- The [Ondat Fencing Controller](https://github.com/storageos/api-manager/tree/master/controllers/fencer) watches for node failures and determines if there are any pods targeted for fencing. 
 
-* The pod must have the label `storageos.com/fenced=true`
-* The pod to be fenced must claim an Ondat volume
-* The Ondat volume claimed by the pod needs to be `online`
+In order for a pod to be fenced, the following criteria listed below is required:
 
-If the node becomes offline and these criteria are met, the pod is deleted and
-rescheduled on another node.
+1. The pod must have the label `storageos.com/fenced=true`,
```

Comment:
can we remove the commas or these bullets?

Reply:
cool, I have added full stops instead.